src/Microsoft.PowerShell.ConsoleGuiTools/ShowObjectTreeCmdletCommand.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+using OutGridView.Models;
+
+namespace OutGridView.Cmdlet
+{
+    [Cmdlet("Show", "ObjectTree")]
+    public class ShowObjectTreeCmdletCommand : PSCmdlet, IDisposable
+    {
+        #region Properties
+        
+        private const string DataNotQualifiedForGridView = nameof(DataNotQualifiedForGridView);
```

Comment:
Is this named correctly? Shouldn't it be "...ForShowObject"?

Reply:
We should be consistent in parameters supported between OCGV and this. I think you should add:

* `-Title`
* `-Filter`
* `-MinUI`
* `-Focus` (if you want to tackle https://github.com/PowerShell/GraphicalTools/issues/127#issuecomment-1520433104)
* `-Search` (https://github.com/PowerShell/GraphicalTools/issues/127#issuecomment-1520433104)

Reply:
I have added in support for 

-Title
-Filter
-MinUI

I have made the filter logic and ux match console grid view (i.e. Regex support with error label).

I think for `-Focus` and `-Search` that if it is not in the base ocgv yet then its not much point implementing it in the tree view at this stage.

Reply:
For example you can now do:

```ps
($A = Get-ChildItem -Path $pshome -Recurse) | Show-ObjectTree -Title "Files in ps home" -MinUI -Filter manifest
```

![image](https://github.com/PowerShell/GraphicalTools/assets/31306100/8f1cfdf2-3270-40ee-8998-6ab6ecf30e9e)